controller/src/main/java/org/jboss/as/controller/SimpleMapAttributeDefinition.java

```diff
@@ -125,9 +127,9 @@ public SimpleMapAttributeDefinition build() {
             if (elementValidator == null) {
                 elementValidator = new ModelTypeValidator(valueType, allowNull, allowExpression);
             }
-            if (attributeMarshaller == null) {
+            /*if (attributeMarshaller == null) {
                 attributeMarshaller = new MapAttributeMarshaller();
-            }
+            }*/
```

Comment:
This looks like a breaking change. There are a number of uses of the other Builder constructors besides the one you change at L108, so if any of those are used attributeMarshaller will be null.